From c88934969fc06b0d875500564157c587ad848a58 Mon Sep 17 00:00:00 2001
From: Jingchang Lu <jingchang.lu@freescale.com>
Date: Fri, 25 Apr 2014 22:15:50 +0800
Subject: [PATCH 063/255] arm: dts: ls1021a-qds: update board specific node
 info

Signed-off-by: Jingchang Lu <jingchang.lu@freescale.com>
[Kevin: Original patch taken from
LS1021A-SDK-V1.1-ARM-SOURCE-20140815-yocto.iso]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm/boot/dts/ls1021a-qds.dts | 47 +++++++++++++++++++++++++++++----------
 1 file changed, 35 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a-qds.dts b/arch/arm/boot/dts/ls1021a-qds.dts
index 680edb7..3596444 100644
--- a/arch/arm/boot/dts/ls1021a-qds.dts
+++ b/arch/arm/boot/dts/ls1021a-qds.dts
@@ -14,7 +14,7 @@
 	model = "LS1021A QDS Board";
 
 	chosen {
-		bootargs = "earlyprintk initrd=initramfs.cpio root=/dev/ram0 rw console=ttyS0,115200";
+		bootargs = "earlyprintk initrd=initramfs.cpio root=/dev/ram0 rw console=ttyS0,115200 user_debug=31";
 	};
 
 	regulators {
@@ -89,11 +89,11 @@
 	};
 };
 
-&duart2 {
+&duart0 {
 	status = "okay";
 };
 
-&duart3 {
+&duart1 {
 	status = "okay";
 };
 
@@ -109,7 +109,7 @@
 
 &enet2 {
 	phy-handle = <&phy2>;
-	 phy-connection-type = "rgmii-id";
+	phy-connection-type = "rgmii-id";
 };
 
 &i2c0 {
@@ -131,6 +131,29 @@
 			};
 		};
 
+		i2c@2 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x2>;
+
+			ina220@40 {
+				compatible = "ti,ina220";
+				reg = <0x40>;
+				shunt-resistor = <1000>;
+			};
+
+			ina220@41 {
+				compatible = "ti,ina220";
+				reg = <0x41>;
+				shunt-resistor = <1000>;
+			};
+
+			adt7461a@4c {
+				compatible = "adt7461a";
+				reg = <0x4c>;
+			};
+		};
+
 		i2c@3 {
 			#address-cells = <1>;
 			#size-cells = <0>;
@@ -160,18 +183,18 @@
 
 &ifc {
 	status = "okay";
-	#address-cells = <1>;
+	#address-cells = <2>;
 	#size-cells = <1>;
 	/* NOR, NAND Flashes and CPLD on board */
-	ranges = <0x0 0x60000000 0x08000000
-		0x2 0x68000000 0x00010000
-		0x3 0x7fb00000 0x00000020>;
+	ranges = <0x1 0x0 0x68000000 0x08000000
+		0x2 0x0 0x7e800000 0x00010000
+		0x3 0x0 0x7fb00000 0x00000020>;
 
-		nor@0 {
+		nor@1 {
 			#address-cells = <1>;
 			#size-cells = <1>;
 			compatible = "cfi-flash";
-			reg = <0x0 0x8000000>;
+			reg = <0x1 0x0 0x8000000>;
 			bank-width = <2>;
 			device-width = <1>;
 
@@ -207,7 +230,7 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			compatible = "fsl,ifc-nand";
-			reg = <0x2 0x10000>;
+			reg = <0x2 0x0 0x10000>;
 
 			partition@0 {
 				/* This location must not be altered  */
@@ -252,7 +275,7 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 			compatible = "fsl,p1010rdb-cpld";
-			reg = <0x3 0x0000020>;
+			reg = <0x3 0x0 0x0000020>;
 			bank-width = <1>;
 			device-width = <1>;
 		};
-- 
2.0.2

